src/Management/test/Endpoint.Test/ManagementPort/ManagementEndpointServedOnDifferentPortTest.cs

```diff
@@ -98,6 +98,37 @@ public async Task AspNetDefaultPort_AlternateManagementPortConfigured_Accessible
         actuatorResponse.StatusCode.Should().Be(HttpStatusCode.OK);
     }
 
+    [Fact]
+    public async Task AspNetDefaultPort_AlternateManagementPortConfigured_AccessibleOnCfInstancePorts()
+    {
+        using var instancePorts = new EnvironmentVariableScope("CF_INSTANCE_PORTS",
+            "[{\"external\":40041,\"internal\":8090,\"external_tls_proxy\":8998,\"internal_tls_proxy\":8999}]");
+
+        const string managementPort = "8090";
+        const string externalTlsProxyPort = "8998";
+        const string internalTlsProxyPort = "8999";
+
+        var appSettings = new Dictionary<string, string?>
+        {
+            // The test listens on the proxy ports in lieu of Cloud Foundry's proxy infrastructure
+            ["urls"] = $"http://*:{externalTlsProxyPort};http://*:{internalTlsProxyPort}",
+            ["management:endpoints:port"] = managementPort
+        };
+
+        await using WebApplication app = await CreateAppAsync(appSettings);
+
+        using HttpClient httpClient = CreateHttpClient();
+
+        HttpResponseMessage actuatorResponse = await httpClient.GetAsync(new Uri($"http://localhost:{managementPort}/actuator"));
+        actuatorResponse.StatusCode.Should().Be(HttpStatusCode.OK);
+
+        actuatorResponse = await httpClient.GetAsync(new Uri($"http://localhost:{internalTlsProxyPort}/actuator"));
+        actuatorResponse.StatusCode.Should().Be(HttpStatusCode.OK);
+
+        actuatorResponse = await httpClient.GetAsync(new Uri($"http://localhost:{externalTlsProxyPort}/actuator"));
+        actuatorResponse.StatusCode.Should().Be(HttpStatusCode.OK);
```

Comment:
Consider a case for `external` here, if that makes sense.

Reply:
We don't currently have a use case for `External`